osu.Game/Screens/Select/BeatmapDetailArea.cs

```diff
@@ -29,6 +29,9 @@ public WorkingBeatmap Beatmap
                 beatmap = value;
                 Leaderboard.Beatmap = beatmap?.BeatmapInfo;
                 Details.Beatmap = beatmap?.BeatmapInfo;
+
+                // Hide detail area if no beatamps available (DummyWorkingBeatmap)
+                this.FadeTo(beatmap is DummyWorkingBeatmap ? 0 : 1);
```

Comment:
add a duration at least. let's go with 200.

Reply:
yeah, we don't wanna go higher than that